FlightReservation/components/screens/ItemsMyFlights.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import {View, Text} from 'react-native';
+import CrossPlatformIcon from 'react-native-cross-platform-icons';
+import MyFlightsStyles from '../assets/styles/MyFlightsStyles';
+
+const ItemsMyFlights = ({item}) => {
+  return (
+    <>
+      <View style={[MyFlightsStyles.row, MyFlightsStyles.spaceBetween]}>
+        <View style={MyFlightsStyles.column}>
+          <Text style={MyFlightsStyles.fontBold}>BEG</Text>
+          <Text style={MyFlightsStyles.fontNormal}>Serbia</Text>
+        </View>
+        <View style={[MyFlightsStyles.column, MyFlightsStyles.centerVertical]}>
+          <CrossPlatformIcon
+            name="airplane"
+            size={25}
+            styles={MyFlightsStyles.centerHorizontal}
+            color="rgba(74,144,226,1)"
```

Comment:
I recomend to change the color of the + icon button and the plane icon from  color="rgba(74,144,226,1)" to 
color ="rgba(92,110,248,1)" which is the title color to have a consisting color palette.

Reply:
I agree with you